### DISCUSSION TITLE
VTV Belarus

### DISCUSSION THREAD

**Discussion by DzuyCuong** [iptv-org/community#453] [Show and tell]
### Channel ID

VTV.by

### Stream URL

http://86.57.244.99:8080/vtv/manifest.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Thank you for the stream: it will be included in the next update.

**Comment by freearhey** [comment]
http://86.57.244.99:8080/vtv/manifest.m3u8 (offline)
<img width="320" alt="image" src="https://user-images.githubusercontent.com/7253922/232242359-86ae52cc-2efb-4842-b6a5-f122925a0a08.png">

**Comment by freearhey** [comment]
http://www.v-tv.by/
![image translated](https://github.com/iptv-org/iptv/assets/7253922/55c0adf4-a171-4217-b52a-4b7da56f87d1)
